### ISSUE TITLE
Fix libnl detection

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1313]
**Reported by clopez on 30 Apr 2014 01:27 UTC**

While working on the packaging of aircrack-ng 1.2-0-beta3 for Debian I noticed the following warning 
```
dpkg-shlibdeps: warning: package could avoid a useless dependency if debian/aircrack-ng/usr/sbin/besside-ng debian/aircrack-ng/usr/sbin/airbase-ng debian/aircrack-ng/usr/sbin/airserv-ng debian/aircrack-ng/usr/sbin/wesside-ng debian/aircrack-ng/usr/bin/wpaclean debian/aircrack-ng/usr/sbin/easside-ng debian/aircrack-ng/usr/sbin/aireplay-ng debian/aircrack-ng/usr/sbin/airodump-ng debian/aircrack-ng/usr/sbin/airtun-ng debian/aircrack-ng/usr/sbin/tkiptun-ng were not linked against libnl-3.so.200 (they use none of the library's symbols)
dpkg-shlibdeps: warning: package could avoid a useless dependency if debian/aircrack-ng/usr/sbin/besside-ng debian/aircrack-ng/usr/sbin/airbase-ng debian/aircrack-ng/usr/sbin/airserv-ng debian/aircrack-ng/usr/sbin/wesside-ng debian/aircrack-ng/usr/bin/wpaclean debian/aircrack-ng/usr/sbin/easside-ng debian/aircrack-ng/usr/sbin/aireplay-ng debian/aircrack-ng/usr/sbin/airodump-ng debian/aircrack-ng/usr/sbin/airtun-ng debian/aircrack-ng/usr/sbin/tkiptun-ng were not linked against libnl-genl-3.so.200 (they use none of the library's symbols)
```

Upon investigation it turns that the code detecting libnl is only defining DCONFIG_LIBNL30 but not DCONFIG_LIBNL ... and a quick look at src/osdep/linux.c shows that defining one without the another don't makes any sense.

**Comment by aircrack-ng** [comment]
**Attachment by clopez on 30 Apr 2014 01:28 UTC**

Uploaded file [fix-libnl-detection.diff](https://github.com/aircrack-ng/trac-attachments/blob/master/1313/20140430-012824_956-fix-libnl-detection.diff)

**Comment by aircrack-ng** [comment]
**Comment by clopez on 30 Apr 2014 01:27 UTC**

With the above patch applied the warning not longer shows all builds as expected.

**Comment by aircrack-ng** [comment]
**Comment by clopez on 30 Apr 2014 01:27 UTC**

Fixed on aircrack-ng/aircrack-ng@758717ce072b8a6345a5ef37395789a880cf2aa7
